### PR TITLE
Billing and subscription freshness

### PR DESCRIPTION
**Pull request by stephaurelio**
## Description

This PR refreshes billing and subscription pages. It also reorganizes the content as new products are being added and management around those products will grow, the Core-focused ToC was quite long. The Core billing and subscription pages have been consolidated under subheadings.

- [Billing deploy preview](https://deploy-preview-19573--docsdocker.netlify.app/billing/)
- [Subscription deploy preview](https://deploy-preview-19573--docsdocker.netlify.app/subscription/)

## Related issues or tickets

Closes JIRA: https://docker.atlassian.net/browse/ENGDOCS-2013
Related to JIRA: https://docker.atlassian.net/browse/ENGDOCS-2008

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by dvdksn — APPROVED** [review]
LGTM

unrelated/question: why is this an image?

image

File: content/subscription/core-subscription/upgrade.md [hunk 1]
```diff
@@ -13,14 +14,20 @@ When you upgrade to a paid subscription, you immediately have access to all the
...
 
 4. From the drop-down menu select **Billing**, then the account you want to upgrade.
 
-5. In the **Plan** section, select **Change plan** and then choose the plan you'd like to upgrade to.
+5. Go to your current plan, then select **Change plan** and then choose the plan you'd like to upgrade to.
+
+   > **Tip**
+   >
+   > If your current plan is a free plan, select **Buy now**.
+   {.tip}
```
Comment (weswhite) [inline content/subscription/core-subscription/upgrade.md:28]: 👌

File: content/subscription/core-subscription/remove-seats.md [hunk 2]
```diff
@@ -29,4 +30,4 @@ For example, if you receive your billing on the 8th of every month for 10 seats
 
 5. The **Billing** tab confirms the seat removal and the details on when the changes take effect.
 
-To cancel the removal of seats, [contact support](https://hub.docker.com/support/contact/).
\ No newline at end of file
+You can cancel the removal of seats before your next billing cycle. To do so, select **Cancel change** in the bottom-right corner.
```
Comment (weswhite) [inline content/subscription/core-subscription/remove-seats.md:33]: might be worth highlighting that the change is scheduled to take effect at the end of the current billing cycle

**Review by weswhite — APPROVED** [review]
looks good

File: content/subscription/core-subscription/remove-seats.md (hunk 2 above)
Comment (stephaurelio) [reply content/subscription/core-subscription/remove-seats.md:33]: We have a little blurb about this at the top of the doc but isn't in the diff 👍

**Pull request merged**